docs/Developers/Writing_Native_Bears.rst

```diff
@@ -130,6 +131,8 @@ the creative name CommunicationBear:
 
 .. code:: python
 
+    import logging
+
     from coalib.bears.LocalBear import LocalBear
```

Comment:
Same here: I've modified the imports' order based on a suggestion from `isort` to improve clarity a little bit :)